java/mapreduce/src/test/org/apache/orc/mapred/TestOrcFileEvolution.java

```diff
@@ -235,35 +235,37 @@ public void testListEvolution() {
   @Test
   public void testPreHive4243CheckEqual() {
     // Expect success on equal schemas
-    checkEvolution("struct<_col0:int,_col1:string>",
+    checkEvolutionPosn("struct<_col0:int,_col1:string>",
                    "struct<_col0:int,_col1:string>",
                    struct(1, "foo"),
-                   struct(1, "foo", null), false, addSarg, false);
+                   struct(1, "foo", null),
+                   false, addSarg, false);
```

Comment:
Let's keep the original form to reduce the patch size.

Reply:
Done